exporter.js

```diff
@@ -74,15 +76,20 @@ const createBody = (meta) => {
   );
 };
 
-const exportAsPR = async (report, token) => {
-  const octokit = new Octokit({auth: `token ${token}`});
+const exportAsPR = async (report, token, octokit) => {
```

Comment:
Rather than passing in both `token` and `octokit`, how about we just always pass in `octokit`? That would mean creating an `Octokit` instance in the /export endpoint, and makes the test and production code more similar. This would be a case of [dependency injection](https://en.wikipedia.org/wiki/Dependency_injection), and is how one often ends up having to structure things for testability in less dynamic languages.

Some of the following code would also be simplified.